README.md

```diff
@@ -55,32 +61,57 @@ After you have installed GOV.UK Frontend the `@govuk-frontend` package will appe
 
 ## Import assets
 
-Import styles into your main Sass file.
+### Import styles
+
+You need to import the GOV.UK Frontend styles into the main Sass file in your project.
 
-To import all components, add the line:
+1. To import all components, add the below to your Sass file:
 ```CSS
 @import "@govuk-frontend/all/all";
 ```
 
-To import an individual component (for example, a button), add the line:
+2. To import an individual component (for example a button), add the below to your Sass file:
 ```CSS
 @import "@govuk-frontend/button/button";
 ```
 
+You should place the above code before your other Sass rules, or Sass imports, if you want to override
```

Comment:
Wouldn't it be better to have this paragraph before the lines of code, after L66?

Reply:
Done!